lu_vp_detect/vp_detection.py

```diff
@@ -546,6 +550,8 @@ def create_debug_VP_image(self, show_image=False, save_image=None):
                         (default=false)
             save_image: Provide a path to save the image to file
                        (default=None - no image is saved)
+            show_vp:    Show the vanishing point on the image
+                        (default = false, added by Harry Beggs)
```

Comment:
Remove `added by Harry Beggs`.  We can see you made these changes in the commit history.